recipes/macsyfinder/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "MacSyFinder" %}
+{% set version = "2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: c4b6a314c9185e6178b02e30d8e2da10380e0a9bdc9c2af78a80994fb083263a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.7,<=3.10
```

Comment:
can you please remove the pins here

Reply:
- python ??
how conda will know which version of python to use

Reply:
it will use all python versions available that are supported... currently 3.7 - 3.10

Reply:
[6cdc25b](https://github.com/bioconda/bioconda-recipes/pull/36684/commits/6cdc25b3a456808215a1183633f6428dc4f89b92)